src/index.tsx

```diff
@@ -21,6 +21,7 @@ import {
 	ListIcon,
 	ListItem,
 	ListSubheader,
+	PickerComponent,
```

Comment:
Why this name?

Reply:
we have used this name because the component itself is using  Picker component from react-native.if we will use picker then it will be stuck in loop as there would be two picker

Reply:
Ok